README.md

```diff
@@ -20,6 +20,9 @@ The next step after provisioning an HPCS instance is to [initialize](https://clo
 For more information, see [components and concepts](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-understand-concepts) of HPCS and [about service instance initialization](https://cloud.ibm.com/docs/hs-crypto?topic=hs-crypto-introduce-service) in the Cloud Docs.
 
 
+If you provision an HPCS instance with a `private-only` endpoint, both `public` and `private` endpoints URLs are included in the output. The public endpoint can be referenced only if there is a need for a temporary switch to enable public endpoint otherwise you can disregard the public endpoint.
```

Comment:
```suggestion
If you provision an HPCS instance with a `private-only` endpoint, both `public` and `private` endpoints URLs are included in the output. You can switch to the public endpoint if needed, but generally you can ignore it.
```

See if you think this is clearer.

Reply:
Hmm, I think if we say directly that `you can switch...` then it generates some more questions related to security.  

Following are the exact words from HPCS team if that helps us to reframe it better.
```
we should document the public endpoint references are there for convenience reason and only valid if public-and-private is setup. There might be usecases where it was setup as private-only, and for a temporary switch to enable public, the endpoint info is already there for applications to be prepared start using it, even before the config change....
```

Reply:
Can we say this - 
```
If you provision an HPCS instance with a `private-only` endpoint, both `public` and `private` endpoints
URLs are included in the output. The public endpoint can be used if there is a need for a temporary switch to enable public endpoint without any configuration change otherwise you can ignore it.
```

Reply:
Ah, I get it. We want to try to use active voice as much as possible (for example, not, `can be used`, but `you can use`). Here's a take with a cautionary note. But go with what you think works best. 

> If you provision an HPCS instance with a `private-only` endpoint, both `public` and `private` endpoints URLs are included in the output. The public endpoint is included for convenience in case you need to switch to it temporarily. However, make sure that you switch back to the private endpoint as soon as possible.